src/lib/components/Label/tests/LabelTest.svelte

```diff
@@ -0,0 +1,14 @@
+<script lang="ts">
+	import { Label } from '$lib/components/Label';
+	import { labelDataTestId, inputId, labelText } from './constants';
+
+	type MouseDownEvent = MouseEvent & {
+		currentTarget: EventTarget & HTMLLabelElement;
+	};
+	export let onMouseDown: (event: MouseDownEvent) => void;
```

Comment:
This sin't being used, right? If so, you can delete it

Reply:
You're right. I'll remove.